lib/easypost/client.rb

```diff
@@ -76,19 +76,8 @@ def make_request(
   )
     response = @http_client.request(method, endpoint, nil, body, api_version)
 
-    status_code = response.code.to_i
-
-    if status_code < 200 || status_code >= 300
-      error = JSON.parse(response.body)['error']
-
-      raise EasyPost::Error.new(
-        error['message'],
-        status_code,
-        error['code'],
-        error['errors'],
-        response.body,
-      )
-    end
+    potential_error = EasyPost::Exceptions::ApiError.handle_api_error(response)
+    raise potential_error unless potential_error.nil?
```

Comment:
thought: let's just call this `error`, that way checking for a nil error makes more sense